utils/duplicate-whitelist-cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
+cat utils/duplicate-whitelist.txt | while read -r line; do
```

Comment:
I prefer the following:

    for line in `cat utils/duplicate-whitelist.txt`; do

More succinct, no need to pipe